charts/hivemq-platform/tests/hivemq_services_test.yaml

```diff
@@ -104,6 +104,58 @@ tests:
                   targetPort: mqtt-1884
```

Comment:
should we update this test to make sure it does not contain any label?

Reply:
Same here, this is already asserted by just expecting a label using `content`.